src/ck-editor.component.ts

```diff
@@ -121,9 +126,8 @@ export class CKEditorComponent implements OnInit, OnDestroy, OnChanges, AfterVie
   }
 
   private destroyCKEditor() {
-    if (this.ckIns) {
-      this.ckIns.removeAllListeners();
```

Comment:
Why remove this code?

Reply:
He removed it to prevent this issue: chymz/ng2-ckeditor#28
I don't know, if it has other impacts, but it at least works for this issue, so i merged it.

Reply:
Ok,thanks